src/views/RequestQueueView.test.js

```diff
@@ -1,85 +1,121 @@
+        type="button"
+        data-testid="dragButton"
+        onClick={() => onDragEnd({ source: { index: 1 }, destination: { index: 0 } })}
+      >
+        Drag Button
+      </button>
+      <span data-testid="rowDraggable">{isRowDraggable({ id: 1 }, 1).toString()}</span>
+    </div>
+  )),
+}));
 jest.mock('../components', () => ({
-  SortableList: jest.fn(() => null),
-  Loading: jest.fn(() => null),
+  Loading: jest.fn(() => <div>Loading...</div>),
+}));
+jest.mock('../utils', () => ({
+  isNotYetFilled: jest.fn(() => true),
+  isPageRequest: jest.fn(() => false),
+}));
+jest.mock('../routes/utils', () => ({
+  getFormattedYears: jest.fn(() => '2023'),
+  getFormattedPublishers: jest.fn(() => 'Publisher'),
+  getFormattedContributors: jest.fn(() => 'Contributors'),
 }));
 
-describe('RequestQueueView', () => {
-  const inProgressRequests = [
-    {
-      status: requestStatuses.AWAITING_DELIVERY,
+const loadmock = false;
+const loadingmock = true;
+const data = {
+  notYetFilledRequests: [
+    { position: 1, title: 'Request 1' },
+    { position: 2, title: 'Request 2' },
+    { position: 3, title: 'Request 3' },
+  ],
+  inProgressRequests: [
+    { id: '3', position: 3, length: 4 },
+  ],
+  request: {
+    instance: {
+      title: 'Test Title',
+      publication: {
+        publisher: 'Test Publisher',
+      },
+      contributorNames: ['Author 1', 'Author 2'],
     },
-    {
-      status: requestStatuses.AWAITING_PICKUP,
-    },
-    {
-      status: requestStatuses.IN_TRANSIT,
-    },
-  ];
-  const notYetFilledRequests = [
-    {
-      status: requestStatuses.NOT_YET_FILLED,
-    },
-  ];
-  const mockedData = {
-    inProgressRequests,
-    notYetFilledRequests,
-  };
-
-  afterEach(() => {
-    FulfillmentRequestsData.mockClear();
-    SortableList.mockClear();
-    Loading.mockClear();
+  }
+};
+const mockOnReorder = jest.fn().mockResolvedValue();
+const mockOnClose = jest.fn();
+const mockisTlrEnabled = true;
+describe('RequestQueueView', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    render(
+      <RequestQueueView
+        data={data}
+        isTlrEnabled={mockisTlrEnabled}
+        onClose={mockOnClose}
+        onReorder={mockOnReorder}
+        isLoading={loadmock}
+      />
+    );
   });
 
-  describe('when all data is already loaded', () => {
-    beforeEach(() => {
-      render(
-        <RequestQueueView
-          data={mockedData}
-          isLoading={false}
-        />
-      );
-    });
-
-    it('should execute `FulfillmentRequestsData` with correct requests collection', () => {
-      const expectedResult = {
-        contentData: inProgressRequests,
-      };
-
-      expect(FulfillmentRequestsData).toHaveBeenCalledWith(expectedResult, {});
-    });
+  it('should perform reload event', () => {
+    const refreshButton = screen.getByRole('button', { name: /ui-requests.requestQueue.refresh/i });
+    expect(refreshButton).toBeInTheDocument();
+    userEvent.click(refreshButton);
```

Comment:
Could you clarify what is the reason to have a click event after `expect` statement?

Reply:
After clicking you should check something. I would suggest creating a separate test where you simulate a 'click' event and expect some result after that event.